vendor/bat-native-ledger/src/bat/ledger/internal/database/database_migration.h

```diff
@@ -15,6 +15,7 @@ namespace ledger {
 class LedgerImpl;
 
 namespace database {
+extern uint32_t test_target_version;
```

Comment:
Can we put this behind a `ForTesting` method? For example, something like:

```cpp
class DatabaseMigration {
  public:
    static void SetTargetVersionForTesting(uint32_t version);
};
```

Reply:
Sure, good idea!